queries.sql

```diff
@@ -8,3 +8,17 @@ SELECT * FROM animals WHERE neutered = true;
 SELECT * FROM animals WHERE name <> 'Gabumon';
 SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
 
+/* day two query and update of the animals table*/
+SELECT COUNT(*) AS total_animals FROM animals;
+SELECT COUNT(*) AS never_tried_to_escape FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) AS average_weight FROM animals;
+SELECT neutered, MAX(escape_attempts) AS max_escape_attempts
+FROM animals
+GROUP BY neutered;
+SELECT species, MIN(weight_kg) AS min_weight, MAX(weight_kg) AS max_weight
+FROM animals
+GROUP BY species;
+SELECT species, AVG(escape_attempts) AS avg_escape_attempts
+FROM animals
+WHERE YEAR(date_of_birth) BETWEEN 1990 AND 2000
```

Comment:
- [ ] it seems that you have not pushed the below part of the requirements to the repository. please make sure to include these.

![image](https://github.com/richie1988/vet_clinic/assets/39240820/a9ed99ca-9d08-4d68-b182-e8f82565e860)

Reply:
Thank you your highlight i thought i did add maybe when i was making changes i made a mistake thank you